device/vlk/vlk_pass.cpp

```diff
@@ -238,22 +238,28 @@ namespace RayGene3D
 
 
       {
+        auto layers = uint32_t(1);
+
         std::vector<VkImageView> rt_attachment_views(rt_views.size());
         for (size_t i = 0; i < rt_views.size(); ++i)
         {
           const auto& rt_view = rt_views[i];
           rt_attachment_views[i] = (reinterpret_cast<VLKView*>(rt_view.get()))->GetView();
-          extent_x = std::max(extent_x, rt_view->GetResource().GetExtentX());
-          extent_y = std::max(extent_y, rt_view->GetResource().GetExtentY());
+          extent_x = std::max(extent_x, rt_view->GetResource().GetExtentX() >> rt_view->GetMipmapOffset());
```

Comment:
Let us add two methods to set up both extent_x/extent_y
We can further validate rt/ds size against these values